app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -101,6 +95,12 @@ class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, ht
       }
 
   private def providerIdQueryParam(providerId: String): Seq[(String, String)] = Seq("providerId" -> providerId)
+
+  private def logSuccessfulExchange[T](`type`: String, payload: T)(implicit fmt: Format[T]): Unit =
+    logger.debug(`type` + "\n" + Json.toJson(payload))
+
+  private def logFailedExchange(`type`: String, exception: Throwable): Unit =
+    logger.error(`type` + " failed", exception)
```

Comment:
Just curious - why error not warn?

Reply:
If I remember correctly, `error` level logs are treated very seriously and might even cause us being called in the middle of the night (?)
I'd rather use `warn` in that case.

Reply:
This happens when the back end returns a non-2xx, in which case we probaly want to know why

Reply:
@MaciejRewera that's only if enough error level logs are raised to reach the PagerDuty threshold

Reply:
As long as I don't have to get out of bed, I approve.

Reply:
On the other hand is this won't be logged on error-handler on top level ?